packages/fxa-auth-server/lib/routes/session.js

```diff
@@ -398,6 +398,20 @@ module.exports = function (
 
         request.emitMetricsEvent('session.resend_code');
 
+        // Check to see if this account has a verified TOTP token. If so, then it should
+        // not be allowed to bypass TOTP requirement by sending a sign-in confirmation email.
+        try {
```

Comment:
We could probably add similar check here https://github.com/mozilla/fxa/blob/56108382558d7d0c3ebc9e4b8341d83032d1bb3e/packages/fxa-auth-server/lib/routes/session.js#L445